merkle_tree/src/main.rs

```diff
@@ -0,0 +1,432 @@
+use sha256::digest;
+
+struct Merkle_Tree {
+    tree: Vec<String>,
+    leaves: usize,
+    og_leaves: Vec<String>,
+}
```

Comment:
This struct should be documentated. Also, in Rust the naming convention is to use camel case for types, so it should be named `MerkleTree`.